spaces/S000115/properties/P000200.md

```diff
@@ -0,0 +1,13 @@
+---
+space: S000115
+property: P000200
+value: false
+  - wikipedia: Seifert–Van_Kampen_theorem#Wedge_sum_of_spaces
+    name: Seifert-Van Kampen theorem on Wikipedia
+---
+
+Consider the subspaces $X_1=X\cap\left(\left[0,\frac{2}{5\pi}\right]\times\mathbb{R}\right)$ and $X_2=X\cap\left(\left[\frac{2}{5\pi},1\right]\times\mathbb{R}\right)$.
+Now $X= X_1\cup X_2$, $X_1 \cap X_2 = \{(\frac{2}{5\pi},1)\}$ and $X_i\backslash\{(\frac{2}{5\pi},1)\}$ is open in $X$.
+Therefore we have 
+$$\pi_1(X) \cong \pi_1(X_1\vee X_2) \cong \pi_1(X_1) * \pi_1(X_2) \cong \pi_1(X_1) * \pi_1(S^1) \cong \pi_1(X_1) * \mathbb{Z}$$
+which is not trivial. Here we used the special case of the Seifert-Van Kampen theorem for wedge sums (See {{wikipedia:Seifert–Van_Kampen_theorem#Wedge_sum_of_spaces}}) and that $\pi_1(S^1) = \mathbb{Z}$ (See {S170|P200}).
```

Comment:
```suggestion
refs:
- zb: "1044.55001"
  name: Algebraic Topology (Hatcher)
---

Let $p,q\in X$ be two consecutive local maxima on the graph of $f(x)=\sin(1/x)$.
The loop $L\subseteq X$ consisting of the line segment between $p$ and $q$ and the portion of the graph of $f$ below it is homeomorphic to a circle $S^1$.
Hence $\pi_1(L)$ is non-trivial, since {S170|P200}.

On the other hand, $L$ is a retract of $X$ (by a map sending points to left of $p$ to $p$ and points to the right of $q$ to $q$).
By Proposition 1.17 of {{zb:1044.55001}}, $\pi_1(L)$ injects into $\pi_1(X)$, which is therefore non-trivial.

```
Easier proof.